src/Conda.jl

```diff
@@ -103,9 +103,55 @@ function _set_conda_env(cmd, env::Environment=ROOTENV)
     env_var["PYTHONIOENCODING"]="UTF-8"
     env_var["CONDARC"] = conda_rc(env)
     env_var["CONDA_PREFIX"] = prefix(env)
+    _set_path(env_var, env)
     setenv(cmd, env_var)
 end
 
+const path_sep = Compat.Sys.iswindows() ? ';' : ':'
+
+function _set_path(env_var, env::Environment)
+    if Compat.Sys.iswindows()
+        # Environment variables are case-insensitive.  `Base.EnvDict`
```

Comment:
Can you file a Julia issue for this?  Seems like it should be fixed in Base.

Reply:
It's just because

```
julia> typeof(copy(ENV))
Dict{String,String}
```

To solve this, I think you need to add a case-insensitive in-memory dict to `Base`.

Reply:
Seems like the easiest fix would be to replace `copy(ENV)` with
```
env_var = Dict(uppercase(k)=>v for (k,v) in ENV
```
(Otherwise the other code in `_set_conda_env` would have to be fixed too.)